src/test/java/gr/cytech/chatreminderbot/rest/ControlIT.java

```diff
@@ -163,6 +163,68 @@ void checkRemindFormatTest() {
                 "\" ,  \"thread\": { \"name\": \"spaces/" + "SPACE_ID" + "\" }}");
     }
 
+    @Test
```

Comment:
this test and the following one differ only on the expected response from the server.
Either one or both of the tests must be wrong